ghmarkdown/ghmarkdown.py

```diff
@@ -72,12 +74,14 @@ def html_from_markdown(markdown):
         exit()
 
 
-def standalone(html):
-    """ Returns complete html document given markdown html """
+def standalone(html, auto=False):
```

Comment:
Because you've made `auto` global in the main function when it gets assigned, it doesn't need to be an argument here.